src/printer/sygus_print_callback.cpp

```diff
@@ -208,5 +208,8 @@ void SygusEmptyPrintCallback::toStreamSygus(const Printer* p,
   }
 }
 
+const std::shared_ptr<SygusEmptyPrintCallback> SygusEmptyPrintCallback::d_empty_pc =
```

Comment:
This should not be initialized statically, instead you should intialize it within getEmptyPc. The reason is that we don't want to create this unless we need it.

Reply:
PTAL as it is now